receiver/azureeventhubreceiver/go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/azuree
 go 1.18
 
 require (
-	github.com/Azure/azure-amqp-common-go/v3 v3.2.3
-	github.com/Azure/azure-event-hubs-go/v3 v3.3.20
+	github.com/Azure/azure-amqp-common-go/v4 v4.0.0
```

Comment:
Does the change to v4 have any implications for end-users of the Azure event hub receiver? Or are the breaking changes limited to the Go API?

Reply:
It does not lead to any changes for end users of the receiver. There were very minor changes to the azure-amqp-common-go interface which meant a version bump was required. See the diff https://github.com/Azure/azure-amqp-common-go/compare/v3.2.3...v4.0.0

Reply:
Alright, yeah, it doesn't look like there are any behavioral changes from what I can tell. Thanks for checking!